test-network-function/identifiers/identifiers.go

```diff
@@ -97,6 +97,11 @@ var (
 		Url:     formTestURL(common.NetworkingTestKey, "icmpv4-connectivity"),
 		Version: versionOne,
 	}
+	// TestICMPv4ConnectivityIdentifierMultus tests icmpv4 connectivity on multus networks.
+	TestICMPv4ConnectivityIdentifierMultus = claim.Identifier{
```

Comment:
I would put the "Identifier" word as suffix always.

Reply:
Makes sense, fixed. Thanks!